frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -87,6 +100,99 @@ class OverrideForkPointDialog(
             thisRowComment
           )
       }
+
+      row {
+        radioButton(
+          format(
+            getString(
+              "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.custom"
+            ),
+            radioButtonComment
+          ),
+          OverrideOption.CUSTOM
+        )
+          .comment(
+            thisRowComment
+          )
+
+        comboBox<ICommitOfManagedBranch?>(
+          (listOf(branch.forkPoint, parentBranch.pointedCommit) + branch.commits.toMutableList()).filterNotNull(),
+          object : DefaultListCellRenderer() {
+            private val defaultBackground = UIManager.get("List.background") as Color
+            override fun getListCellRendererComponent(
+              list: JList<*>?,
+              value: Any?,
+              index: Int,
+              isSelected: Boolean,
+              cellHasFocus: Boolean
+            ): Component {
+              val commit: ICommitOfManagedBranch? = value as ICommitOfManagedBranch?
+              componentOrientation = list!!.componentOrientation
```

Comment:
Hmm what's the guarantee that `list` is non-null? 🤔

Reply:
we're the one constructing it initially with the list of commits.